src/languages/markdown.coffee

```diff
@@ -18,7 +18,7 @@ module.exports = {
     "md"
   ]
 
-  defaultBeautifier: "Tidy Markdown"
+  defaultBeautifier: "Remark"
```

Comment:
I think you make a good argument for defaulting to Remark 👍 . However, first need to address the changes in `expected` above.